code/edge/src/androidTest/java/com/adobe/marketing/mobile/CompletionHandlerFunctionalTests.java

```diff
@@ -79,14 +80,17 @@ public void setup() throws Exception {
 		MobileCore.registerExtensions(Arrays.asList(Edge.EXTENSION, Identity.EXTENSION), o -> latch.countDown());
 		latch.await();
 		assertExpectedEvents(false);
-		resetTestExpectations();
+		resetTestExpectations(mockNetworkService);
```

Comment:
If resetTestExpectations is in some base class and we are not instantiating networkService in that base class, I would not pass networkService to it. We could have a reset method in the test class.

```
reset(){
resetMockNetworkService
resetTestExpectation
}
```